keel-core/src/main/kotlin/com/netflix/spinnaker/keel/exceptions/ExportError.kt

```diff
@@ -0,0 +1,3 @@
+package com.netflix.spinnaker.keel.exceptions
+
+class ExportError(message: String) : RuntimeException(message)
```

Comment:
Is this translated into an HTTP error on the export API?

Reply:
Do you think this should be a `500`? this is basically just used because kotlins null safety didn't catch that I throw an exception when there's an empty list, but then finding the first in the list can still be null. I don't think we will ever throw it, and if we don't I think it's a bug on our side.

Reply:
In that case 500 seems right. I was more just wondering if this exception might start getting used for other things in future.

Reply:
I don't think it will, but if it does I'll add the exception handling by the controllers.